concepts/05 UI Components/Button/00 Getting Started with Button/00 Getting Started with Button.md

```diff
@@ -0,0 +1,7 @@
+The Button is the UI component that performs an action when a user clicks it. This tutorial shows how to add the Button to a page, apply styling, and configure the core features. As a result, you will get the following UI component:
```

Comment:
I wouldn't start with the button definition. You can remove it and start with the second sentence. It's just as clear. 

In general, I'd replace the word "get" with another verb, if I can. Books, such as "Misplaced Modifier" refer to "get" as a "weak and boring" verb. 

 ```suggestion
This tutorial shows how to add a Button to a page, apply styling, and configure its core features. As a result, you will create the following UI component:
```